lnrpc/rpc.proto

```diff
@@ -1498,6 +1498,13 @@ message ChannelEdge {
 }
 
 message ChannelGraphRequest {
+     /**
+     Whether unannounced channels should be returned or not. If set,
+     unannounced channels are returned.
+     Unannoucned channels are both private channels and public
+     channels whose authentication proof has not been confirmed yet.
```

Comment:
suggestion: ~~whose authentication proof has not been confirmed yet~~ that are not yet announced to the network